src/spikeinterface/core/sparsity.py

```diff
@@ -119,6 +125,87 @@ def unit_id_to_channel_indices(self):
                 self._unit_id_to_channel_indices[unit_id] = channel_inds
         return self._unit_id_to_channel_indices
 
+    def sparsify_waveforms(self, waveforms: np.ndarray, unit_index: int) -> np.ndarray:
```

Comment:
Is there a reason to go for `unit_index`? For high-level functions we usually use `unit_id` (same for the `unit_id_to_channel...` dictionaries).

Reply:
OK, let me changed this. I was just chosing the simplest type but what you mention makes sense.